pinot-tools/src/main/resources/examples/stream/meetupRsvp/meetupRsvp_realtime_table_config.json

```diff
@@ -7,7 +7,9 @@
     "segmentPushType": "APPEND",
     "segmentAssignmentStrategy": "BalanceNumSegmentAssignmentStrategy",
     "schemaName": "meetupRsvp",
-    "replication": "1"
+    "replication": "1",
```

Comment:
no need to change this file?

Reply:
adding retentionTimeUnit is helpful. I was annoyed by the NPE thrown by the RetentionManager when debugging the quickStart.